go.sum

```diff
@@ -43,6 +42,8 @@ github.com/gammazero/deque v0.0.0-20200227231300-1e9af0e52b46 h1:iX4+rD9Fjdx8Skm
 github.com/gammazero/deque v0.0.0-20200227231300-1e9af0e52b46/go.mod h1:D90+MBHVc9Sk1lJAbEVgws0eYEurY4mv2TDso3Nxh3w=
 github.com/gammazero/workerpool v0.0.0-20200311205957-7b00833861c6 h1:1Cy/haf7XO4OyrkGid0Wq5CMluIErbvDptVAt8UTy38=
 github.com/gammazero/workerpool v0.0.0-20200311205957-7b00833861c6/go.mod h1:/XWO2YAUUpPi3smDlFBl0vpX0JHwUomDM/oRMwRmnSs=
+github.com/gauravgahlot/tink v0.0.0-20200806080035-1ab6de4bab5c h1:lJ8Tl+jk1pvzmB+GQKUJHcFPswUNLEARqk4JunWtXg8=
+github.com/gauravgahlot/tink v0.0.0-20200806080035-1ab6de4bab5c/go.mod h1:5Kv1re+s/uSfE3vWsoOS+tKwZYwLrrPXgEXViTbEmv4=
```

Comment:
I believe this will removed once we have my PR merged. :thinking:

Reply:
yes, let's have your tink PR merged, then e can update this.